src/abseil-cpp/preprocessed_builds.yaml.gen.py

```diff
@@ -0,0 +1,130 @@
+#!/usr/bin/env python2.7
```

Comment:
Let's not force new code to a version of Python that will be officially EOL in less than a month. Either `#!/usr/bin/env python3` or no shebang at all.

Reply:
Changed it to have python3 shebang.